### PR TITLE
ul: add Chinese translation

### PR DESCRIPTION
**Pull request by legendqi**
- [ ] The page (if new), does not already exist in the repository.
- [ ] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [ ] The page has 8 or fewer examples.
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [ ] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages.zh/linux/ul.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# ul
+
+> 执行文本的下划线。
+> 给定字符串中的每个字符必须单独加下划线。
+> 更多说明请访问： <https://manned.org/ul>。
```
Comment (marchersimon) [inline pages.zh/linux/ul.md:5]:
```suggestion
> 更多说明请访问： <https://manned.org/ul>.
```
We use half-width punctuation afer Latin characters

**Review by marchersimon — COMMENTED** [review]
Thanks @legendqi, just left one suggestion below

File: pages.zh/linux/ul.md (hunk 1 above)
Comment (legendqi) [reply pages.zh/linux/ul.md]: had fixed

**Comment by marchersimon** [comment]
@blueskyson or @BillLucky would you maybe want to have a look?
@legendqi, there's no need to force-push, since we squash all commits automatically before merging

**Comment by CleanMachine1** [comment]
the file input should be made into `path/to/file.txt`

**Comment by legendqi** [comment]
Do I need to close the pull request and restart the request?

**Comment by legendqi** [comment]
@CleanMachine1 sorry,I don't understand what you mean, file path?

File: pages.zh/linux/ul.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# ul
+
+> 执行文本的下划线。
+> 给定字符串中的每个字符必须单独加下划线。
+> 更多说明请访问： <https://manned.org/ul>.
+
+- 在适用的情况下用下划线显示文件的内容：
+
+`ul {{file.txt}}`
```
Comment (einverne) [inline pages.zh/linux/ul.md:9]:
```suggestion
ul {{path/to/file.txt}}
```

File: pages.zh/linux/ul.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+- 用由破折号组成的下划线显示文件的内容 `-`：
+
+`ul -i {{file.txt}}`
```
Comment (einverne) [inline pages.zh/linux/ul.md:13]:
```suggestion
ul -i {{path/to/file.txt}}
```

**Review by einverne — CHANGES_REQUESTED** [review]
Thank you for the translation. There is no need to use backtick here. And when we refer to a file, we often add path/to.

File: pages.zh/linux/ul.md (hunk 2 above)
Comment (navarroaxel) [reply pages.zh/linux/ul.md:9]: Please translate the `path/to/file` for get this easier for non-English speakers.

**Comment by CleanMachine1** [comment]
> Thank you for the translation. There is no need to use backtick here. And when we refer to a file, we often add path/to.

The backticks are our syntax for the command

File: pages.zh/linux/ul.md (hunk 3 above)
Comment (legendqi) [reply pages.zh/linux/ul.md]: fixed,please check.

File: pages.zh/linux/ul.md (hunk 2 above)
Comment (legendqi) [reply pages.zh/linux/ul.md]: @navarroaxel fixed.please check

File: pages.zh/linux/ul.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# ul
+
+> 执行文本的下划线。
+> 给定字符串中的每个字符必须单独加下划线。
+> 更多说明请访问： <https://manned.org/ul>.
```
Comment (navarroaxel) [inline pages.zh/linux/ul.md:5]:
```suggestion
> 更多信息见： <https://manned.org/ul>.
```

File: pages.zh/linux/ul.md [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+
+- 在适用的情况下用下划线显示文件的内容：
+
+`ul 文件路径`
```
Comment (navarroaxel) [inline pages.zh/linux/ul.md:9]:
```suggestion
`ul {{文件路径}}`
```

File: pages.zh/linux/ul.md [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+
+- 用由破折号组成的下划线显示文件的内容 `-`：
+
+`ul -i 文件路径`
```
Comment (navarroaxel) [inline pages.zh/linux/ul.md:13]:
```suggestion
`ul -i {{文件路径}}`
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]

File: pages.zh/linux/ul.md (hunk 4 above)
Comment (blueskyson) [reply pages.zh/linux/ul.md:5]:
```suggestion
> 更多信息： <https://manned.org/ul>.
```

File: pages.zh/linux/ul.md [hunk 7]
```diff
@@ -0,0 +1,13 @@
+# ul
+
+> 执行文本的下划线。
+> 给定字符串中的每个字符必须单独加下划线。
+> 更多信息： <https://manned.org/ul>.
+
+- 在适用的情况下用下划线显示文件的内容：
+
+`ul {{ 文件路径 }}`
```
Comment (navarroaxel) [inline pages.zh/linux/ul.md:9]: Why the spaces?
```suggestion
`ul {{文件路径}}`
```

**Review by navarroaxel — APPROVED** [review]

**Comment by einverne** [comment]
> > Thank you for the translation. There is no need to use backtick here. And when we refer to a file, we often add path/to.
> 
> The backticks are our syntax for the command

Oh, sorry, I forget about it.

**Review by einverne — APPROVED** [review]
Looks good now.

**Review by marchersimon — APPROVED** [review]
Thanks @legendqi

**Pull request merged**